assets/src/edit-story/components/form/numeric.js

```diff
@@ -131,6 +131,11 @@ function Numeric({
   const placeholder = isMultiple ? __('multiple', 'web-stories') : '';
 
   const handleChange = useCallback((event) => {
+    // If the user types something in, clear timeout
+    // so onChange doesn't get triggered with possibly
+    // old value.
+    window.clearTimeout(submitOnChangeTimeout.current);
```

Comment:
Should this first check if the timeout is set and clear only if yes?

Reply:
I think it's okay to call regardless.  According to the mozilla docs it's okay to pass in an invalid ID, I assume they store them in a hash of some sort so the lookup would be quick (just as quick as an if statement).  Plus, I'd then have to add in more logic to make sure I set `submitOnChangeTimeout.current` to `-1` or something after each use (so more code).

From the docs:
```
Passing an invalid ID to clearTimeout() silently does nothing; no exception is thrown.
```

Reply:
Hah, was it different at some point in history? For some reason, I remember this requiring a value (`null` being invalid) but looks like not. Thanks for the info!